libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -921,13 +920,12 @@ AP_InertialSensor::init(uint16_t loop_rate)
     for (uint8_t i=0; i<get_gyro_count(); i++) {
         // only allocate notches for IMUs in use
         if (_use[i]) {
-            if (_harmonic_notch_filter.enabled()) {
```

Comment:
make it depend on notch enable || fft enable?

Reply:
Fixed